CHANGELOG.md

```diff
@@ -1,11 +1,25 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
+This CHANGELOG follows [this format](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).
 
 ## [Unreleased]
 
-## [1.0.1] - 2017-07-29
+## [3.0.0] - 2018-11-20
+### Breaking Change
```

Comment:
Should be in the plural form I am guessing there is some ambiguity in the docs as in one place it refers to `The Breaking Change` but over in the example we use it in the plural [form](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#breaking-changes). I have opened https://github.com/sensu-plugins/community/issues/122 to resolve the documentation ambiguity.